src/app/modules/project/overview/overview.component.scss

```diff
@@ -171,8 +204,9 @@
   }
 
   .option-select {
-    margin-bottom: 15px;
+    margin-bottom: 10px;
     width: 100%;
+    color: rgba(#323232, 0.7) ;
```

Comment:
Instead of using the color code, try using the available variable found in _variables.scss